buildscripts/kokoro/linux_artifacts.sh

```diff
@@ -5,6 +5,19 @@ if [[ -f /VERSION ]]; then
   cat /VERSION
 fi
 
-readonly GRPC_JAVA_DIR=$(cd $(dirname $0)/../.. && pwd)
+readonly GRPC_JAVA_DIR="$(cd "$(dirname "$0")"/../.. && pwd)"
 
-# A place holder at the moment
+rm -rf /tmp/source_head/protobuf
+mkdir -p /tmp/source_head/protobuf
```

Comment:
The "set -e" on line 2 makes it so that if any commands fail, the entire script exits with a failure. The other options are also good defensive features that we turn on.